.travis.yml

```diff
@@ -6,6 +6,6 @@ os:
 after_success: npm run coverage
 
 node_js:
-  - "4"
   - "6"
+  - "8"
```

Comment:
I've been using the following config recently to not have to worry about keeping this list up to date... however, we do technically support one LTS version back (v6, in this case). It would be nice if there was an alias for that but I don't know of one.

```
- "lts/*"
- "stable"
```

Reply:
I think testing of the oldest version we support is worth it, node.js major release cycle is long enough that adding new versions when needed isn't a big deal.

Reply:
I think I'm with @coreyfarrell on this one, I like that it's more explicit.